cookiecutter/cli.py

```diff
@@ -48,7 +51,7 @@ def validate_extra_context(ctx, param, value):
 
     # Convert tuple -- e.g.: (u'program_name=foobar', u'startsecs=66')
     # to dict -- e.g.: {'program_name': 'foobar', 'startsecs': '66'}
-    return dict(s.split('=', 1) for s in value) or None
+    return dict(s.split('=', 1) for s in value) or None  # type: ignore
```

Comment:
This is sub-optimal, but relies on a fix here: https://github.com/python/typeshed/issues/256